src/Core/Application/Security/ProviderConfiguration/OpenId/UseCase/FindOpenIdConfiguration/FindOpenIdConfigurationResponse.php

```diff
@@ -104,4 +106,41 @@ class FindOpenIdConfigurationResponse
      * @var boolean
      */
     public bool $verifyPeer;
+
+    /**
+     * @var boolean
+     */
+    public bool $isAutoImportEnabled;
```

Comment:
this class is badly initialized, variables of type bool have no default value

Reply:
public properties can be accessed only when they are initialized, so I'm not sure it's mandatory to set a default value